pom.xml

```diff
@@ -15,6 +15,8 @@
   <properties>
     <findbugs-maven-plugin.version>3.0.4</findbugs-maven-plugin.version>
     <findbugs.failOnError>true</findbugs.failOnError>
+    <java.level>6</java.level>
+    <java.level.test>6</java.level.test>
```

Comment:
Probably better to default to `${java.level}`; or simply omit, as differentiating them currently does not work in IDEA so other developers would complain.